guides/v2.2/extension-dev-guide/view-models.md

```diff
@@ -0,0 +1,66 @@
+---
+group: php-developer-guide
+subgroup: 99_Module Development
```

Comment:
The following metadata is no longer necessary; please remove:

- `subgroup:` 
- ` menu_title:`
- `menu_order:`